From 7c18afb0d02de40da3aa20540516b01fddf904df Mon Sep 17 00:00:00 2001
From: Didier Raboud <odyx@debian.org>
Date: Mon, 11 Sep 2017 10:46:39 +0200
Subject: Use CMAKE_INSTALL targets

---
 CMakeLists.txt | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 148c747..33758f3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -76,10 +76,11 @@ if(BUILD_EXAMPLES)
 endif()
 
 # Installation
+include(GNUInstallDirs)
 
 INSTALL(FILES ${HEADERS} DESTINATION include/kitchensink/)
 INSTALL(TARGETS SDL_kitchensink SDL_kitchensink_static
-    RUNTIME DESTINATION bin
-    LIBRARY DESTINATION lib
-    ARCHIVE DESTINATION lib
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
 )
